zhaquirks/philips/__init__.py

```diff
@@ -188,7 +179,12 @@ class PhilipsRemoteCluster(CustomCluster):
         PressType(SHORT_RELEASE, COMMAND_M_SHORT_RELEASE),
     ]
 
-    button_press_queue = ButtonPressQueue()
+    def __init__(self, endpoint, is_server=True):
+        """Initialize button press queue for each button."""
+
+        super().__init__(endpoint, is_server)
+
+        self.button_press_queue = {k: ButtonPressQueue() for k in self.BUTTONS}
```

Comment:
Let's just remove the two empty lines here.
```suggestion
    def __init__(self, endpoint, is_server=True):
        """Initialize button press queue for each button."""
        super().__init__(endpoint, is_server)
        self.button_press_queue = {k: ButtonPressQueue() for k in self.BUTTONS}
```